src/irr/asset/IAssetManager.cpp

```diff
@@ -230,26 +230,65 @@ void main()
 }
 		)===",asset::ISpecializedShader::ESS_FRAGMENT,"irr/builtin/materials/lambertian/singletexture/specializedshader");
 
+        // uv debug shader
+        buildInShader(R"===(
+        #version 430 core
+        layout(location = 0) in vec3 vPos;
+        layout(location = 2) in vec2 vTexCoord;
+
+        #include <irr/builtin/glsl/vertex_utils/vertex_utils.glsl>
+        #include <irr/builtin/glsl/broken_driver_workarounds/amd.glsl>
 
+        layout (set = 1, binding = 0, row_major, std140) uniform UBO 
+        {
+            irr_glsl_SBasicViewParameters params;
+        } CamData;
+        
+        layout(location = 0) out vec2 uv;
+
+        void main()
+        {
+            gl_Position = irr_glsl_pseudoMul4x4with3x1(irr_builtin_glsl_workaround_AMD_broken_row_major_qualifier_mat4x4(CamData.params.MVP), vPos);
+	        uv = vTexCoord;
+        }
+		)===", asset::ISpecializedShader::ESS_VERTEX, "irr/builtin/materials/lambertian/uv_debug_shader/specializedshader");
+        buildInShader(R"===(
+        #version 430 core
+
+        layout(set = 0, binding = 0) uniform sampler2D albedo;
+
+        layout(location = 0) in vec2 uv;
+
+        layout(location = 0) out vec4 pixelColor;
+
+        void main()
+        {
+            pixelColor = texture(albedo, uv);
+        }
+		)===", asset::ISpecializedShader::ESS_FRAGMENT, "irr/builtin/materials/lambertian/uv_debug_shader/specializedshader");
```

Comment:
ekhm what's the one with the texture doing here?

Reply:
You wanted me to create 3 debug shaders, should I remove it?

Reply:
they don't need a texture declared